frontend/playwright.config.ts

```diff
@@ -1,7 +1,7 @@
 import type { PlaywrightTestConfig } from '@playwright/test';
 import dotenv from 'dotenv';
 
-dotenv.config();
+dotenv.config({ path: './.env' });
```

Comment:
oh nice, if we do this we can do the `.env.test` thing above?

Reply:
I don't think so, see above reasoning.